ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -125,7 +125,7 @@ class FaultInjector
   void checkAntFaults();
   void checkCamFaults();
 
-  ///////publishers and subsscribers
+  ///////publishers and subscribers
```

Comment:
two back slashes is all you need
```suggestion
  // publishers and subscribers
```